packages/ember/lib/index.js

```diff
@@ -13,13 +13,13 @@ import 'ember-routing-views';
 import Ember from 'ember-metal/core';
 import { runLoadHooks } from 'ember-runtime/system/lazy_load';
 
-if (Ember.__loader.registry['ember-template-compiler']) {
+if (Ember.__loader.registry['ember-template-compiler/index']) {
```

Comment:
Ya, I'll update the requireModule part but I don't think require exposed the registry itself (though we could do that).